pyartcd/pyartcd/pipelines/brew_scan_osh.py

```diff
@@ -0,0 +1,111 @@
+"""
+[STORES STATE IN WORKSPACE]
+This job scans the candidate tags for a particular version, and triggers scans for builds that are tagged into it.
+"""
+import click
+import json
+import base64
+import yaml
+import os
+from ghapi.all import GhApi
+from pyartcd import exectools
+from pyartcd.runtime import Runtime
+from pyartcd.cli import cli, pass_runtime, click_coroutine
+
+FILE_PATH = f"{os.getenv('WORKSPACE')}/event.json"
+
+
+class OshScan:
+    def __init__(self, runtime: Runtime, email: str, version: str):
+        self.runtime = runtime
```

Comment:
we can add support from dry_run, using `self.runtime.dry_run`

Reply:
fixed in https://github.com/openshift-eng/art-tools/pull/51/commits/906d84751f15b00b02083b9cfe428b05c0888896